docs/source/architecture.rst

```diff
@@ -137,188 +133,96 @@ This namespace has, as in the case of middletier namespace, allocated pool of
+<https://argoproj.github.io/argo-workflows/>`__. Additional logic used during
+executing workflows is taken from `thoth-station/workflow-helpers repository
+<https://github.com/thoth-station/workflow-helpers>`__.
 
 Graph Namespace
 ###############
 
-A separate namespace was created for database related deployments. Components
-running in this namespace:
-
-* Thoth's knowledge graph - a `PostgreSQL database <https://www.postgresql.org/>`_
-
-* ``pgbouncer`` - a `PostgreSQL bouncer <https://www.pgbouncer.org/>`_ to recycle
-  and manage connections to the database - all the components talk to this component
-  when a query is performed
-
-* ``pgweb`` - a `web interface <https://sosedoff.github.io/pgweb/>`_ to interact with
-  Thoth's knowledge graph
+A separate namespace for database related deployments.
 
-* ``postgresql-metrics-exporter`` - PostgreSQL related metrics (see Grafana
-  dashboards section) for the main database observability
+**Components running in this namespace:**
 
-Thamos
-######
-
-`Thamos <https://thoth-station.ninja/docs/developers/thamos>`_ is a CLI tool
-created for end-users of Thoth. Thamos offers a simple command line interface
-to consume Thoth's advises (recommendations) and Thoth's provenance checks both
-done against data stored in the knowledge graph. The tool packs in an
-automatically generated Python client library from User API OpenAPI
-specification The library can be used to integrate with Thoth. See `Thamos
-repository for documentation and an example usage
-<https://thoth-station.ninja/docs/developers/thamos>`_.
+* Thoth's knowledge graph - a `PostgreSQL database <https://www.postgresql.org/>`_
 
-Kebechet
-########
+* `pgbouncer <https://www.pgbouncer.org/>`__ - recycle and manage connections
+  to the database; all the components talk to this component rather than
+  directly to PostgreSQL
 
-Another consumer of Thoth's data is a bot called `Kebechet
-<https://github.com/thoth-station/kebechet>`_ that operates directly on
-repositories on hosted on GitHub or GitLab and it opens pull requests or issues
-automatically for users. See :ref:`integration documentation for more
-info <integration>`.
+* `pgweb <https://sosedoff.github.io/pgweb/>`__ (optional) - interact with
+  Thoth's knowledge graph via UI
 
-Build Watcher
-#############
+* `postgresql-metrics-exporter
+  <https://github.com/wrouesnel/postgres_exporter>`__ - PostgreSQL related
+  metrics for the the database observability
 
-This component was designed to automatically watch for cluster events - wait
-for OpenShift builds to finish and submit the resulting container image
-produced to Thoth for analysis. As the container image registry is usually
-cluster internal, without any route exposed to the world, build-watcher can
-also cooperate with external registries. Documentation can be found in `the
-README file in the GitHub project
-<https://github.com/thoth-station/build-watcher>`_.
+* `graph-backup-job
```

Comment:
this is currently in the frontend, but as you mentioned it would be more suitable in this namespace.

Reply:
this would take care of this https://github.com/thoth-station/thoth-application/issues/1166